XWebView/XWVHttpConnection.m

```diff
@@ -341,7 +341,8 @@ - (void)stream:(NSStream *)aStream handleEvent:(NSStreamEvent)eventCode {
     NSString *name;
 
     int class = (int)response.statusCode / 100 - 1;
-    assert(class >= 0 && class < 5);
+    NSCAssert(class >= 0 && class < 5, @"<XWV> status code must be in the range [0, 500)");
```

Comment:
According to [RFC7231 section 6](https://tools.ietf.org/html/rfc7231#section-6), status code should be in range [100, 599].